src/game.cpp

```diff
@@ -8513,17 +8513,22 @@ bool game::get_liquid_target( item &liquid, item *const source, const int radius
                               const monster *const source_mon,
                               liquid_dest_opt &target )
 {
-    if( !liquid.made_of( LIQUID ) ) {
+    if( item::find_type( liquid.typeId() )->phase == SOLID ) {
```

Comment:
You can still access the type of an item directly, it has a `type` member, you don't need to look this up.